lms/static/sass/base/_grid-settings.scss

```diff
@@ -4,10 +4,13 @@
 $max-width: 1200px;
 /* Override the default global box-sizing */
 $border-box-sizing: false;
+/* Grid width variables */
+$large-min-width: 769px;
```

Comment:
I don't think this variable is needed. It seems to be referenced here, in the ``$large`` breakpoint, and in one particular spot (https://github.com/edx/edx-platform/pull/7548/files#diff-ed9911acad8061e8e3605c66f1d569f4R155). Can we just use the ``$large`` breakpoint instead without this?